src/Router.ts

```diff
@@ -439,8 +439,19 @@ export class Router<C extends Context> extends Evented implements RouterInterfac
 		return this._outletContextMap.has(outletId);
 	}
 
-	getOutlet(outletId: string): OutletContext | undefined {
-		return this._outletContextMap.get(outletId);
+	getOutlet(outletId: string | Array<String>): OutletContext | undefined {
+		if (Array.isArray(outletId)) {
+			const outletContextArray = arrayFrom(this._outletContextMap);
+			const outlets = arrayFrom(outletId);
```

Comment:
Why do you need to `arrayFrom` here? We are already know it is an array as we are in the `Array.isArray(outletId)` block.

Reply:
Ah good catch, left over code from when I was trying to convert the "needle" we are trying to find into an array, even if it was one already. Removed!